src/WebSocket.cs

```diff
@@ -185,10 +185,9 @@ private async Task ProcessSendQueue()
                         }
                     }
 
-                    var o = new MemoryStream();
-                    var w = new BinaryWriter(o);
-                    new ClientMessage.BSATN().Write(w, message);
-                    await Ws!.SendAsync(o.ToArray(), WebSocketMessageType.Binary, true, CancellationToken.None);
+                    var messageBSATN = new ClientMessage.BSATN();
+                    var encodedMessage = IStructuralReadWrite.ToBytes(messageBSATN, message);
+                    await Ws!.SendAsync(encodedMessage, WebSocketMessageType.Binary, true, CancellationToken.None);
```

Comment:
do the current tests exercise this codepath, or should it be tested via BitCraft?

Reply:
Unfortunately, the tests don't use networking, they're pure unit tests. I don't expect this to be a functional change as that helper does the same thing, but I guess worth testing.